docs/docs/importing-media-content.md

```diff
@@ -8,7 +8,7 @@ title: Importing Media Content
 
 - [Image graphics can be imported](/docs/importing-assets-into-files/) with Webpack and queried with GraphQL.
 - Images can also be [included from the `static folder`](/docs/static-folder/).
-- SVG content can be embedded into JSX. Here's a [handy JSX converter](https://transform.now.sh/html-to-jsx/).
+- SVG content can be embedded into JSX. Here's a [handy JSX converter](https://transform.tools/).
```

Comment:
In case they change the default selection:

```suggestion
- SVG content can be embedded into JSX. Here's a [handy JSX converter](https://transform.tools/svg-to-jsx).
```

Reply:
Nice! Thanks!